### PR TITLE
snmp-ups: support Eaton Automatic Transfer Switch ATS30

### PR DESCRIPTION
**Pull request by thalman**
ATS30 MIB differs from previously integrated EATON ATS driver.

### CODE REVIEW

File: drivers/eaton-ats30-mib.c [hunk 1]
```diff
@@ -0,0 +1,370 @@
+/* eaton-ats30-mib.c - subdriver to monitor eaton_ats30 SNMP devices with NUT
+ *
+ *  Copyright (C)
+ *  2011 - 2017	Tomas Halman <TomasHalman@eaton.com>
```
Comment (jimklimov) [inline drivers/eaton-ats30-mib.c]: Does the copyright extend that long? When did you start working on this? ;)

File: drivers/eaton-ats30-mib.c [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+	/* enterprises.534.10.1.3.5.0 = INTEGER: 37937541 */
+	{ "device.uptime", 0, 1, ".1.3.6.1.4.1.534.10.1.3.5.0", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.6.0 = INTEGER: 284 */
+	/* { "unmapped.atsMessureTransferedTimes", 0, 1, ".1.3.6.1.4.1.534.10.1.3.6.0", NULL, SU_FLAG_OK, NULL, NULL }, */
+	/* enterprises.534.10.1.3.7.0 = INTEGER: 4 */
+	{ "input.source", 0, 1, ".1.3.6.1.4.1.534.10.1.3.7.0", NULL, SU_FLAG_OK, ats30_source_info, NULL },
+    
+    /* atsStatus */
+    /* ========= */
+#if 0
```
Comment (jimklimov) [inline drivers/eaton-ats30-mib.c]: Should those `#if 0` blocks stay? If there is a possibility of uncommenting them later (e.g. when more research/discovery/support is done, then at least a comment is IMHO due to explain what the hard ifdef is for and why the code is just not deleted.

File: drivers/snmp-ups.c [hunk 3]
```diff
@@ -85,6 +86,7 @@ static mib2nut_info_t *mib2nut[] = {
 	&eaton_ats,
 	&apc_ats,
 	&raritan_px2,
+        &eaton_ats30,
```
Comment (jimklimov) [inline drivers/snmp-ups.c]: Tab vs space? Indentation difference is visible...

File: drivers/eaton-ats30-mib.c (hunk 1 above)
Comment (aquette) [inline drivers/eaton-ats30-mib.c]: Tom probably just generated using my script and replaced the name, not touched the year.
BTW, the copyright is Eaton, and Tom is the author... will fix when merging the PR...

File: drivers/snmp-ups.c (hunk 3 above)
Comment (aquette) [inline drivers/snmp-ups.c]: will also fix over the merge...

File: drivers/eaton-ats30-mib.c (hunk 2 above)
Comment (aquette) [inline drivers/eaton-ats30-mib.c]: will add a note over the merge...
BTW, there seem to have been an issue when doing the generation. The comment line theoretically state the resolved OID (i.e. textual mode, not numeric), when the needed MIBs are available...

**Pull request merged**